test/system/500-networking.bats

```diff
@@ -760,4 +760,11 @@ EOF
     done
 }
 
+@test "podman network rm --force bogus" {
+    run_podman 1 network rm bogus
+    is "$output" "Error: unable to find network with name or ID bogus: network not found" "Should print error"
+    run_podman network rm --force bogus
+    is "$output" "" "Should print no output"
```

Comment:
That is not how Docker behaves:
```
~ $ sudo docker rm -f bogus
Error: No such container: bogus
```

Reply:
If we don't care about this compat, users could just use `--ignore` and we don't need this PR.

Reply:
But I am not married to printing the error.  If others feel strongly about it, feel free to merge.  In the general case we should remain compatible where possible - sometimes bug compatible.

Reply:
> If we don't care about this compat, users could just use `--ignore` and we don't need this PR.

No right no we exit with code > 0. The goal is to exit with 0, so basically --force should imply --ignore IMO.

Reply:
IIRC we decided --force not imply --ignore when adding --ignore as we were afraid of breaking backwards compat with existing Podman users.  But I also think --force should imply it ... just as so many other tools do.